From 8b4ff97413fc32ea7f817586bc682ff2cc34527b Mon Sep 17 00:00:00 2001
From: Siddhesh Poyarekar <siddhesh@redhat.com>
Date: Mon, 10 Mar 2014 13:17:54 +0530
Subject: [PATCH] Mark nscd service as forking in systemd service file (BZ
 #16639)

Currently the nscd service is installed in systemd as a simple
service, which means that it is able to handle its own errors and does
not quit.  Since nscd does not fit that description, i.e. it can exit
on errors like, say, failing to parse nscd.conf, it should be declared
as forking instead.
---
 nscd/nscd.service | 4 ++--
 3 files changed, 8 insertions(+), 3 deletions(-)

diff --git a/nscd/nscd.service b/nscd/nscd.service
index 99c7563..b7428a3 100644
--- a/nscd/nscd.service
+++ b/nscd/nscd.service
@@ -4,8 +4,8 @@
 Description=Name Service Cache Daemon
 
 [Service]
-Type=simple
-ExecStart=/usr/sbin/nscd --foreground
+Type=forking
+ExecStart=/usr/sbin/nscd
 ExecStop=/usr/sbin/nscd --shutdown
 ExecReload=/usr/sbin/nscd -i passwd
 ExecReload=/usr/sbin/nscd -i group
-- 
1.9.0

